tools/bpf_inspect.py

```diff
@@ -64,15 +71,7 @@ def get_linked_func(bpf_prog):
     return f"{linked_prog_id}->{linked_btf_id}: {kind.name} {linked_name}"
 
 
-def get_tramp_progs(bpf_prog):
-    try:
-        tr = bpf_prog.aux.member_("trampoline")
-    except LookupError:
-        # Trampoline is available since Linux kernel commit
-        # fec56f5890d9 ("bpf: Introduce BPF trampoline") (in v5.5).
-        # Skip trampoline if current kernel doesn't support it.
-        return
-
+def get_prog_from_tramp(trampoline):
```

Comment:
This potentially yields multiple programs, so maybe this should be named `get_progs_from_tramp` or something like that?